doc/classes/Node.xml

```diff
@@ -623,10 +623,11 @@
 			</return>
 			<argument index="0" name="node" type="Node">
 			</argument>
-			<argument index="1" name="keep_data" type="bool" default="false">
+			<argument index="1" name="keep_groups" type="bool" default="false">
 			</argument>
 			<description>
 				Replaces a node in a scene by the given one. Subscriptions that pass through this node will be lost.
+				When [code]keep_groups[/code] is [code]true[/code], the [code]node[/code] will be added to the same groups that the replaced node is in.
```

Comment:
```suggestion
				If [code]keep_groups[/code] is [code]true[/code], the [code]node[/code] is added to the same groups that the replaced node has been in.
```

Reply:
Sorry, should've used the suggestions feature from the start. I hope it's clear now what I mean with the tense.

Reply:
Just a quick question, could 'has been in' confuse someone into thinking the 'old' node is no longer in those groups?

Reply:
I just went from your original use of "was". You can use "is" if the old node is still in those groups.

Reply:
Ok, force push incoming :smile: